api/users/models.py

```diff
@@ -121,6 +121,12 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(BEFORE_DELETE)
+    def delete_orphan_organisations(self):
```

Comment:
Having thought about this, I'm not sure that this is definitely the functionality that we want. There are circumstances in which users can be deleted where they will not be notified that they are also deleting organisations. Let's discuss.

Reply:
The corrected version implements the delete orphan organisations in a view.